Userland/Applications/FileManager/DirectoryView.cpp

```diff
@@ -570,18 +586,7 @@ void DirectoryView::setup_actions()
     });
 
     m_open_terminal_action = GUI::Action::create("Open &Terminal Here", Gfx::Bitmap::load_from_file("/res/icons/16x16/app-terminal.png"), [&](auto&) {
-        posix_spawn_file_actions_t spawn_actions;
-        posix_spawn_file_actions_init(&spawn_actions);
-        posix_spawn_file_actions_addchdir(&spawn_actions, path().characters());
-        pid_t pid;
-        const char* argv[] = { "Terminal", nullptr };
-        if ((errno = posix_spawn(&pid, "/bin/Terminal", &spawn_actions, nullptr, const_cast<char**>(argv), environ))) {
-            perror("posix_spawn");
-        } else {
-            if (disown(pid) < 0)
-                perror("disown");
-        }
-        posix_spawn_file_actions_destroy(&spawn_actions);
+        spawn_terminal (path().characters());
```

Comment:
The String->char*->String roundtrip isn't nice, let's make `spawn_terminal` take a StringView instead, and omit `.characters()` here.
Also, no space after identifier and before the open paren.